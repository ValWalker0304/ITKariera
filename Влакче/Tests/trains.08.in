Add 82658 Kulata F 35
Add 73031 Plovdiv F 6
Add 75784 Tulovo F 28
Add 39447 Shumen P 2
Add 31564 Plovdiv F 36
History
History
Add 60157 Yambol P 8
Travel
Next
Next
Add 37333 Yambol P 2
History
Add 11227 Montana P 3
Next
Add 86655 Vratsa F 23
Add 52112 Tulovo F 8
Travel
Travel
History
History
Travel
Next
Add 27551 Vratsa P 8
Travel
Travel
History
Travel
Next
History
Travel
Travel
History
Add 17973 Shumen F 37
Next
End