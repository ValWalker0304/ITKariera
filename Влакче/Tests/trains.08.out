31564 Plovdiv F 36
75784 Tulovo F 28
75784 Tulovo F 28
31564 Plovdiv F 36
75784 Tulovo F 28
11227 Montana P 3
37333 Yambol P 2
37333 Yambol P 2
11227 Montana P 3
31564 Plovdiv F 36
37333 Yambol P 2
11227 Montana P 3
31564 Plovdiv F 36
60157 Yambol P 8
39447 Shumen P 2
27551 Vratsa P 8
39447 Shumen P 2
39447 Shumen P 2
27551 Vratsa P 8
60157 Yambol P 8
37333 Yambol P 2
11227 Montana P 3
31564 Plovdiv F 36
52112 Tulovo F 8
86655 Vratsa F 23
52112 Tulovo F 8
39447 Shumen P 2
27551 Vratsa P 8
60157 Yambol P 8
37333 Yambol P 2
11227 Montana P 3
31564 Plovdiv F 36
86655 Vratsa F 23
75784 Tulovo F 28
75784 Tulovo F 28
86655 Vratsa F 23
52112 Tulovo F 8
39447 Shumen P 2
27551 Vratsa P 8
60157 Yambol P 8
37333 Yambol P 2
11227 Montana P 3
31564 Plovdiv F 36
17973 Shumen F 37
